src/sandbox/wiktor/LineChart.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { useRef } from 'react';
+import {
+  LineChart,
+  Line,
+  XAxis,
+  YAxis,
+  CartesianGrid,
+  Tooltip,
+  Legend,
+  ResponsiveContainer,
+} from 'recharts';
+
+interface ILineChartProps {
+  data: { datetime: string; value: number }[];
+}
+
+const LineChartComponent: React.FC<ILineChartProps> = ({ data }) => {
+  const chartRef = useRef<HTMLDivElement>(null);
+
+  const minValue = Math.min(...data.map((d) => d.value));
+  const maxValue = Math.max(...data.map((d) => d.value));
+  const margin = 0.02;
+  const yDomain = [
+    Math.round(minValue * (1.0 - margin)),
+    Math.round(maxValue * (1.0 + margin)),
+  ];
+
+  return (
+    <div ref={chartRef} style={{ height: '300px' }}>
+      <ResponsiveContainer width="100%" height="100%" debounce={1}>
+        <LineChart data={data} margin={{ bottom: 60, left: 20, right: 20 }}>
```

Comment:
Odstępy dookoła Charta zostawiłbym do Container'a

```suggestion
        <LineChart data={data} margin={{ bottom: 60, right: 10 }}>
```